contracts/Everlong.sol

```diff
@@ -173,4 +224,51 @@ contract Everlong is Admin, ERC4626, Positions {
     function _afterDeposit(uint256, uint256) internal override {
         // revert("TODO");
     }
+
+    // TODO: Include consideration for idle liquidity.
+    /// @dev Open the maximum amount of positions possible
+    ///      with the contract's current balance.
+    function _openPositions() internal {
+        // Obtain the current balance of the contract.
+        // If the balance is less than hyperdrive's min tx amount, return.
+        // If the balance is greater, use it all to open longs.
+        uint256 _currentBalance = IERC20(_asset).balanceOf(address(this));
+        uint256 _minTxAmount = IHyperdrive(_hyperdrive)
+            .getPoolConfig()
+            .minimumTransactionAmount;
+        if (_currentBalance <= _minTxAmount) return;
```

Comment:
this gets double checked in canRebalance() when executed in the rebalance() function scope

Reply:
Yeah there's probably a better way to do this. My thinking was that we still need to tho because `canRebalance` could return true for only a mature position, and we shouldn't purchase longs